tracer/test/Datadog.Trace.Tests/Propagators/MultiSpanContextPropagatorTests.cs

```diff
@@ -691,11 +692,11 @@ public void TraceContextPrecedence_ExtractedState_WhenMissingDD_OnTracestate(boo
         }
 
         [Theory]
-        [InlineData(false, false)]
-        [InlineData(false, true)]
-        [InlineData(true, false)]
-        [InlineData(true, true)]
-        public void TraceContextPrecedence_ConsistentBehaviour_WithDifferentParentId(bool extractFirst, bool w3CHeaderFirst)
+        [InlineData(false, false, "0123456789abcdef", 987654321)]
```

Comment:
nit: Instead of declaring the values for the lastParentId and updatedSpanId, can you calculate them inside the test method with `if/else` statements or ternary operations?

The value of having the InlineData with two bools and listing all possible inputs as we do before is that we get a really good idea of what to expect by reading the test case name and the parameters to the test case, plus we know we've exhausted all possible inputs. If you add these ID's into the InlineData declaration, I can't immediately tell what the significance is

Reply:
Updated in: https://github.com/DataDog/dd-trace-dotnet/pull/5721/commits/cad9105eeb83f02e61795bce7994f0b48c89d4ac